File: Java/Search/LinearSearch.java
package Java.Search;

public class LinearSearch {
    public static void main(String[] args) {
        
        int[] array = {0, 5, 13, 19, 22, 41, 55, 68, 72, 81, 98};
        
        System.out.println(linearSearch(array, 55));

    }   
    
    public static boolean linearSearch(int[] numbers, int number) {

        for (int i = 1; i < numbers.length; i++) {
            if (numbers[i] == number) {
                return true;
            }
        }

        return false;

    }

}

File: Kotlin/Search/LinearSearch.kt
package Kotlin.Search

fun main() {
    val nums = listOf<Int>(0, 5, 13, 19, 22, 41, 55, 68, 72, 81, 98)
    println(linearSearch(nums, 41))
}

fun linearSearch(numbers: List<Int>, num: Int): Boolean {
    for (i in numbers) {
        if (i == num) {
            return true
        }
    }
    return false
}